### DISCUSSION TITLE
Is it possible to change the login / logout pages per Provider / Application?

### DISCUSSION THREAD

**Discussion by GD-Dal** [goauthentik/authentik#11483] [Q&A]
Hi.
I would like to change the login / logout pages for Immich when people are logging in to it via Authentik.
Is this possible?
Branding seems to do the trick, but then you have to fiddle around with an extra FQDN entry for each Application you set up.
It seems unnecessary.
So is there an easier way?

Thanks

**Comment by BeryJu** [comment]
Login pages can be changed by using the authentication flow on the provider. Logout pages are not yet changeable, this will be possible with https://github.com/goauthentik/authentik/pull/5048

**Comment by GD-Dal** [comment]
Thanks for answering.
It seem that some adjustments can be made there, thanks.
But what about text, logo etc?

![image](https://github.com/user-attachments/assets/393e5c88-7244-4f0c-b8e0-903032d96d5d)

Thanks again

**Reply by BeryJu** [reply]
Logos can be changed via Brands https://docs.goauthentik.io/docs/brands
